### PR TITLE
add additional unit tests demonstrating delegation flows

### PR DESCRIPTION
**Pull request by MikeHathaway**
# Description of change
## High level
* <DESCRIP_OF_CHANGE>
  * <DESCRIP_OF_SUBCHANGE>

# Description of bug or vulnerability and solution
* <PARAGRAPH_EXP_OF_VULN_BUG>
* <PARAGRAPH_EXP_OF_HOW_CHANGE_SOLVES_VULN_OR_BUG>

# Contract size
## Pre Change
<PASTE_OUTPUT_HERE>
## Post Change
<PASTE_OUTPUT_HERE>

# Gas usage
## Pre Change
<PASTE_OUTPUT_HERE>
## Post Change
<PASTE_OUTPUT_HERE>

### CODE REVIEW

File: test/unit/AjnaToken.t.sol [hunk 1]
```diff
@@ -235,6 +235,106 @@ contract AjnaTokenTest is Test {
...
+        assertEq(_token.balanceOf(delegate1), 0);
+        // check voting power
+        assertEq(_token.getVotes(delegator), 1_001 * 1e18);
+        assertEq(_token.getVotes(delegate1), 0);
+    }
+
+    function testMultipleDelegationDifferentDelegatee() external {
+        address delegator = address(2222);
+        address delegate1 = address(3333);
+        address delegate2 = address(4444);
```
Comment (EdNoepel) [inline test/unit/AjnaToken.t.sol:288]: What's the role of `delegate2` in this test?  Nobody delegates to this actor, and we confirm their voting power is 0.

**Review by EdNoepel — APPROVED** [review]
I don't understand what `testMultipleDelegationDifferentDelegatee` is testing, but the first test is quite helpful.

File: test/unit/AjnaToken.t.sol (hunk 1 above)
Comment (ith-harvey) [reply test/unit/AjnaToken.t.sol:288]: Yep, same question as Ed

**Review by ith-harvey — APPROVED** [review]
LGTM

**Pull request merged**